perma_web/frontend/lib/store.js

```diff
@@ -12,14 +13,57 @@ export const getLinksRemainingStatus = (linksRemaining, isNonpaying) => {
     return globalStore.updateLinksRemainingStatus('unlimited_paid')
 }
 
-export const getUserTypes = (isIndividual) => {
-    let types = []
-
+export const getUserTypes = (isIndividual, user) => {
     if (isIndividual) {
-        types = types.concat('individual')
+        return globalStore.updateUserTypes('individual')
+    }
+
+    let userTypes = []
+
+    const topLevelFolders = user.top_level_folders
+    const isOrgAffiliated = topLevelFolders.some(folder => !!folder.organization)
+    const isSponsored = topLevelFolders.some(folder => folder.is_sponsored_root_folder)
+
+    if (isOrgAffiliated) {
+        userTypes = userTypes.concat('orgAffiliated')
+    }
+
+    if (isSponsored) {
+        userTypes = userTypes.concat('sponsored')
+    }
+
+    if (userTypes.length) {
+        return globalStore.updateUserTypes(userTypes)
+    }
+}
+
+export const getOrganizationFolders = async () => {
+    const { data, error, errorMessage } = await useFetch('/api/v1/organizations', {
+        limit: 300,
+        order_by: 'registrar, name'
+    })
```

Comment:
I observe that this method `getOrganizationFolders`, and the name of the variable in the global store `this.organizationFolders`, refers to "organization folders" rather than just "organizations", even though this is a call to the organization API route, but that the same pattern is followed for the naming of `getSponsoredFolders` and `this.sponsoredFolders`. Do you think the similarity of their names suggests that they retrieve and store data of the same type/shape? Would `getUserOrganizations` maybe be a good way to describe what this method is doing?

Reply:
This is a great question. I renamed the function to `getUserOrganizations` and renamed the global store-related state and update method to match that naming in [c017fa7](https://github.com/harvard-lil/perma/pull/3507/commits/c017fa75b5f6edf32916c32d332595bcce2e0316)